src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -144,7 +168,10 @@
         (if @(rf/subscribe [::loading?])
           [[spinner/big]]
           [[roles/when roles/show-admin-edit-buttons?
-            [to-create-catalogue-item]
-            [status-flags/display-archived-toggle #(rf/dispatch [::fetch-catalogue])]
+            [:div.commands.text-left.pl-0
+             [ create-catalogue-item-button]
```

Comment:
Extra space.

Reply:
> I think the selected items should be reset when entering the catalogue page.
> 
> Also, seem that you can select items from multiple non-overlapping searches, which might be ok as a design decision (e.g. if it's not easy to find all items to change with a single search), but it becomes confusing when they are still selected after visiting other pages. Resetting the selection on entering the catalogue page should fix that.

It would prevent the user from viewing specific rows while trying to select the right ones because the list resets on every navigation. Like you can read from commit messages I chose to explicitly allow multiple filters as a feature and disallow archive toggle here.